rbac/management/role/serializer.py

```diff
@@ -49,6 +49,18 @@ def validate_attributeFilter(self, value):
             message = f"attributeFilter operation must be one of {ALLOWED_OPERATIONS}"
             error = {key: [_(message)]}
             raise serializers.ValidationError(error)
+        else:
+            values = value.get("value")
+            if type(values) is str and op == "in":
```

Comment:
These checks are insufficient, as these check only for string. If for example `int` was supplied, it would fall through. Why not check the expected type for each operator?

Be aware, that previously a documentation mentioned a string of comma separated values. Have you checked current state of the database? If users would have already stored badly formatted data, they could hit these errors.

I suggest to prefer `isinstance` over a strong type check. See [Duck Typing](https://docs.python.org/3/glossary.html#term-duck-typing).